src/cdn/ck/createCKCdnBaseBundlePack.ts

```diff
@@ -68,7 +72,30 @@ export function createCKCdnBaseBundlePack(
 
 		// Pick the exported global variables from the window object.
 		checkPluginLoaded: async () =>
-			waitForWindowEntry( [ 'CKEDITOR' ] )
+			waitForWindowEntry( [ 'CKEDITOR' ] ),
+
+		// Check if the CKEditor base bundle is already loaded and throw an error if it is.
+		beforeInject: () => {
+			const installationInfo = getCKBaseBundleInstallationInfo();
+
+			switch ( installationInfo?.source ) {
+				case 'npm':
+					throw new Error(
+						'CKEditor 5 is loaded from NPM. The migration to CKEditor 5 CDN installation is required. ' +
+						`Check the migration guide for more details: ${ createCKDocsUrl( 'updating/migration-to-cdn/vanilla-js.html' ) }`
+					);
+
+				case 'cdn':
+					if ( installationInfo.version !== version ) {
+						throw new Error(
+							`CKEditor 5 version ${ installationInfo.version } is already loaded from CDN. ` +
+							`The requested version is ${ version }.`
```

Comment:
```suggestion
							`CKEditor 5 is already loaded from CDN in version ${ installationInfo.version }. ` +
							`Remove the old <script> and <link> tags loading CKEditor 5 to allow loading the ${ version } version.`
```

@gorzelinski Thoughts?

Reply:
From the language standpoint - correct. And it's more descriptive. I like it 👍